spec/services/list_manuals_service_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "spec_helper"
+require "list_manuals_service"
+require "ostruct"
```

Comment:
Minor: You don't appear to be using `OpenStruct` in the spec, so I think this `require` statement is unnecessary.